code/script.js

```diff
@@ -33,19 +37,290 @@ const showMessage = (message, sender) => {
+    inputWrapper.innerHTML =
+    `<div class="input wrapper" id="input-wrapper">
+      <form id="name-form">
+        <button id="green-btn" class="green-btn" type="submit">Green</button>
+        <button id="blue-btn" class="blue-btn" type="submit">Blue</button>
+        <button id="red-btn" class="red-btn" type="submit">Red</button>
+      </form>
+     </div>
+    `
+  }
+//3 colour options
+  document
+    .getElementById('green-btn')
+    .addEventListener('click', (event) => {
+      event.preventDefault()
+      showMessage("Green", "user");
+      setTimeout(() => sightQuestion("green"), 1200);
+    })
+
+  document
+    .getElementById('blue-btn')
+    .addEventListener('click', (event) => {
+      event.preventDefault()
+      showMessage("Blue", "user")
+      setTimeout(() => sightQuestion("blue"), 1200);
+    })
+
+  document
+    .getElementById('red-btn')
+    .addEventListener('click', (event) => {
+      event.preventDefault()
+      showMessage("Red", "user")
+      setTimeout(() => sightQuestion("red"), 1200);
+    }) 
+}
+
+//A function that uses color-choice to ask question which activates SIGHT
+const sightQuestion = (color) => {
+  showMessage(`Good choice! Look around you. How many ${color} things can you see?`, "bot");
+  inputWrapper.innerHTML = `      
+    <div class="input-wrapper" id="input-wrapper">
+      <form id="number-form">
+        <input id="number-input" type="number"/>
+      <button id="sight-btn" class="send-btn" type="submit">
+        Send
+      </button>
+    </form>
+  </div>
+  `  
+  const numberInput = document.getElementById("number-input")
+
+  //event listener for number input
+  document
+  .getElementById("sight-btn")
+  .addEventListener('click', (event) => {
+    event.preventDefault()
+    const number = numberInput.value
+    showMessage(`${number}`, "user")
+    numberInput.value = ""
+    setTimeout(() => hearingQuestion(`${number}`, `${color}`), 1200)
+  })
+  }
+  
+//A function that reacts to number of sights, and asks question that activates SOUND
+const hearingQuestion = (numberOfSights, color) => {
+  if (numberOfSights == 0) {
+    showMessage(`That's okay. Maybe you can find some sounds instead. Take a moment to listen carefully. How many different sounds do you hear right now?`, "bot")
+  } 
+  else if (numberOfSights > 3) {
+    showMessage(`${numberOfSights}? That's a lot of ${color} things!`, "bot")
+    setTimeout(() => showMessage("Now, take a moment to listen carefully. How many different sounds do you hear?", "bot"), 900)
+  }
+  else {
+    showMessage(`Good. Now, take a moment to listen carefully. How many different sounds do you hear?`, "bot")
+  }
+  inputWrapper.innerHTML = `
+   <div class="input-wrapper" id="input-wrapper">
+    <form id="number-form">
+      <input id="number-input" type="number"/>
+    <button id="sound-btn" class="send-btn" type="submit">
+     Send
+    </button>
+    </form>
+   </div>
```

Comment:
This inputWrapper.innerHTML I can't fully "grasp" right now - but I hope you can explain in a Team session short in the future?
It's probably simple and easy for those who knows it by heart - but I don't yet and now I'm working hard not to "Google my way through my first every Code Review... ;) 
Please bare with me :D